system_helpers/apt/extract.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import logging
+import pathlib
+import re
+import subprocess
+import tempfile
+import typing
+
+import typeguard
+
+from system_helpers.apt import install
+
+
+# https://askubuntu.com/questions/37156/how-to-restore-a-single-file-from-a-package
+# https://askubuntu.com/questions/1512765/extract-one-particular-file-from-a-deb-package-file
+
+@typeguard.typechecked
+def parse_args() -> argparse.Namespace:
+    """
+    Parse CLI arguments.
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--package', type = str, required = True)
+    parser.add_argument('--files',   nargs='+')
+
+    return parser.parse_args()
+
+@typeguard.typechecked
+def candidate(*, package : str) -> str:
+    """
+    Get candidate version for `package`.
+    """
+    output = subprocess.check_output(['apt', 'policy', package]).decode()
+    return re.search(
+        pattern = r'Candidate: ([a-z0-9.\-~+]+)',
+        string  = output,
+    ).group(1)
+
+@typeguard.typechecked
+def main(*, package : str, files : typing.List[str]) -> None:
+    """
+    Download `.deb` for `package` and install only the specified `files`.
+    """
+    version = candidate(package = package)
+
+    logging.info(f'Candidate version for {package} is {version}.')
+
+    install.install_packages(
+        update = True,
+        clean = False,
+        args = ['--download-only'],
+        packages = [package],
+    )
+
+    deb = pathlib.Path('/var/cache/apt/archives/') / f'{package}_{version}_amd64.deb'
+
+    if not deb.is_file():
+        raise FileNotFoundError(deb)
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # List files in the archive. We'll look for the data part.
+        cmd = ['ar', 'tv', deb]
+
```

Comment:
You could delete this empty line too. The comment would then appear with the code block that looks for the data part.